playbooks/roles/aws_logs_agent/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Download the AWS Logs Agent python script
+  get_url:
+    url: "{{ AWS_LOGS_AGENT_SCRIPT_URL }}"
+    dest: /tmp/aws-logs-agent.py
+    mode: "1777"
+  tags:
+    - install
+    - install:base
+
+- name: Copy aws_logs_agent.conf template
+  template:
+    src: "aws_logs_agent.conf.j2"
+    dest: "/etc/aws_logs_agent.conf"
+    mode: "0664"
+
+- name: Run AWS Logs Agent script
+  shell: "python2.7 /tmp/aws-logs-agent.py -n --region {{ AWS_EC2_INSTANCE_REGION }} -c /etc/aws_logs_agent.conf"
```

Comment:
Python 2 has been removed from Ubuntu 20.04. Shouldn't we use Python 3 here?

Also, we can use the built-in [script module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/script_module.html).

Reply:
This script only works for `v2.6 < python > v3.6`, I found the servers either have v2.7 or v3.8, I'll check if I can download python v3.5 and run this script.

Reply:
steps to download python3.5
>Also, we can use the built-in script module.

I had tried script module, but it wasn't working correctly with the sudo rights and was constantly raising permission issues, so I tried shell and it worked fine (followed some existing roles structure.)

Reply:
> can we avoid installing an additional interpreter for this?

@0x29a we already have some steps which requires py3.5 ([ref](https://github.com/openedx/configuration/blob/4a98e09fa8a388cdcf627299e7c9db4bbddc3748/playbooks/roles/common_vars/defaults/main.yml#L141))

I'm following the official guide to installing and setup awslogs-agent ([ref](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/QuickStartEC2Instance.html)) and the script does all the required installation/setup with one command.

I'm looking into how I can use the .deb package to configure the agent to publish logs, I'll update the thread once I found a way to do it!

Reply:
I've found a way to use the Debian package, will update the changes and get it ready for review!

Reply:
@0x29a I've updated the role to use .deb package